Bio/SeqFeature.py

```diff
@@ -851,6 +851,21 @@ def __iter__(self):
             for i in range(self._start, self._end):
                 yield i
 
+    def __eq__(self, other):
+        """Implement equality by comparing all the location attributes."""
+        if not isinstance(other, FeatureLocation):
+            return False
+        return self._start == other.start and \
+            self._end == other.end and \
+            self._strand == other.strand and \
+            self.ref == other.ref and \
+            self.ref_db == other.ref_db
+
+    def __ne__(self, other):
+        """Implement the not-equal operand."""
+        # This is needed for py2, but not for py3.
+        return not (self == other)
```

Comment:
I've used the same style the `Seq` object `_ne_` function used that you linked me to on Twitter.